src/sdk/bucketspace_fs.js

```diff
@@ -121,11 +122,22 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             };
 
             bucket.name = new SensitiveString(bucket.name);
-            bucket.bucket_owner = new SensitiveString(bucket.bucket_owner);
+            // Backward compatibility will take longer without bucket owner name
+            const account_config = await this.config_fs.get_identity_by_id(
+                bucket.owner_account,
+                TYPES.ACCOUNT,
+                { silent_if_missing: true }
+            );
+
+            if (!account_config) {
+                dbg.warn(`Bucket Owner does not exist ${bucket.owner_account}`);
+            }
+            bucket.bucket_owner = new SensitiveString(account_config?.name);
```

Comment:
if the `account_config` is undefined then you wrap the undefined value, is it what you want?

Reply:
initially I threw an error if the owner account config does not exist, but later on I saw it doesn't make sense to fail on that, so since we expect a sensitive string I need to wrap something.. do you have a better suggestion?

Reply:
No, I want to make sure this is what you planned.
I suggest to add a comment in the code about this.